### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by xiokd**
Code pair: @JessicaH46

Azure Link: https://calm-sand-08b0dd810.1.azurestaticapps.net/

### CODE REVIEW

**Review by BenagainB — COMMENTED** [review]
I tested your link and the web app is deploying well. It looks like you have the game page running correctly. If you want, you can add a link in the menu at the top left of the landing page by adding it to the file in layouts/default.vue

**Comment by NazariyPogo** [comment]
Will review

**Comment by NazariyPogo** [comment]
Looking through all the requirements, looks like everything is in, the URL for the site should be in a README but Meg said having it in the comment is fine so no issue there. Looks like it is connected to GitHub with no issues. Clean code. Good job!

File: wordle-web/pages/game.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template>
+    <v-card>
+        <v-card-title>
+            A Creative Title
```
Comment (dcox2005) [inline wordle-web/pages/game.vue:4]: I feel like we were meant to come up with an actual game name here.  Could be wrong though.

**Review by dcox2005 — COMMENTED** [review]
Overall, your website is up and running and you have met the main requirements.  My only thought is the creative title should maybe have what you think a game name should be.

**Comment by mmwoodfo** [comment]
Good job making an Assignment 1 branch for your changes. Great git branch management! Keep that up for each assignment. :)

**Pull request closed without merging**